services/movie_session.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime
+
+from typing import Optional
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```

Comment:
No need to have those objects
Instead of defining them pass theirs ids
```
movie_id=movie_id,
cinema_hall_id=cinema_hall_id
```

Reply:
when I done that - I get error:

FAILED tests/test_main.py::test_movie_session_service_create_movie_session - ValueError: Cannot assign "2": "MovieSession.cinema_hall" must be a "CinemaHall" instance.

and my pytest dont pass

Reply:
so I cannot create MovieSession using id's = I should use CinemaHall instances , am I right ?